layouts/ucf-events-modern.php

```diff
@@ -45,7 +45,7 @@ function ucf_events_display_modern_title( $content, $items, $args, $display_type
 // Modern - main loop
```

Comment:
We'll also need to add fallback message support to the modern_nodesc layout: https://github.com/UCF/UCF-Events-Plugin/blob/master/layouts/ucf-events-modern.php#L157